jenkins/ci.suse.de/cloud-mediacheck.yaml

```diff
@@ -41,6 +41,7 @@
       - shell: |
           export automationrepo=~/github.com/SUSE-Cloud/automation
           export jtsync=${automationrepo}/scripts/jtsync/jtsync.rb
+          name=$(echo ${JOB_NAME} | cut -d "/" -f 1)
```

Comment:
This looks like a workaround for a different issue. If JOB_BASE_NAME is not set, then JOB_NAME also is not set (compare current jobs). So the issue is a more general one.
Lets first find out how to solve this before adding such workarounds.

Btw. I will plan a downtime for an upgrade to the latest version of jenkins, as there were other reports about similar issues.